shared/templates/rsyslog_logfiles_attributes_modify/bash.template

```diff
@@ -0,0 +1,110 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
```

Comment:
missing debian and ubuntu platforms, perhaps easier to set it to multi_platform_all

Reply:
Good point :bow: , added in [79ef5fb](https://github.com/ComplianceAsCode/content/pull/9789/commits/79ef5fb0b1b8cf273869fb251e696fefe3aeefee)